Hmm... this is very strange, I'm getting:
```
CCACHE_CPP2=1 ccache clang++  -DRELEASE -DGIT_VERSION -DLOCALIZE -ffast-math -Os  -Wall -Wextra  -g  -fsigned-char -Werror -std=c++11 -MMD -MP -m32 -I/usr/include/lua5.2 -DLUA -D_GNU_SOURCE -I/usr/include/ncursesw -c src/calendar.cpp -o obj/calendar.o
src/calendar.cpp:319:26: error: use of undeclared identifier 'to_weeks'
        const int week = to_weeks<int>( d );
                         ^
src/calendar.cpp:319:38: error: expected '(' for function-style cast or type construction
        const int week = to_weeks<int>( d );
                                  ~~~^
2 errors generated.
Makefile:787: recipe for target 'obj/calendar.o' failed
make: *** [obj/calendar.o] Error 1
make: *** Waiting for unfinished jobs....
zsh: exit 2     DEBUG_SYMBOLS=yes Platform=Linux Graphics=Curses
DEBUG_SYMBOLS=yes Platform=Linux Graphics=Curses   15.81s user 1.03s system 122% cpu 13.712 total
```

...but `to_weeks` is declared in `calendar.h`, right where `to_days` is (and that one works just fine). Any idea wtf? I'm not really a C++ person, and this feels like a C++ oddity.

--- a/src/calendar.h
+++ b/src/calendar.h
@@ -257,6 +257,8 @@ constexpr T to_hours( const time_duration duration );
 template<typename T>
 constexpr T to_days( const time_duration duration );
 template<typename T>
+constexpr T to_weeks( const time_duration duration );
+template<typename T>
 constexpr T to_moves( const time_duration duration );
 
 template<typename T>

--- a/src/game_inventory.cpp
+++ b/src/game_inventory.cpp
@@ -393,7 +393,7 @@ class comestible_inventory_preset : public inventory_selector_preset
                     g->u.get_skill_level( skill_survival ) >= 4 ) {
                     const islot_comestible item = get_edible_comestible( loc );
                     if( item.spoils > 0 ) {
-                        const std::string freshness = get_freshness( loc );
+                        std::string freshness = get_freshness( loc );
                         return freshness;
                     }
                 }
@@ -406,7 +406,7 @@ class comestible_inventory_preset : public inventory_selector_preset
                     const islot_comestible item = get_edible_comestible( loc );
                     if( item.spoils > 0 ) {
                         if( !get_comestible_item( loc ).rotten() ) {
-                            const std::string time_left = get_time_left_rounded( loc );
+                            std::string time_left = get_time_left_rounded( loc );
                             return time_left;
                         }
                     }
@@ -515,23 +515,22 @@ class comestible_inventory_preset : public inventory_selector_preset
             const double relative_rot = item->is_food_container() ? item->contents.front().get_relative_rot() :
                                         item->get_relative_rot();
             const time_duration shelf_life = get_edible_comestible( loc ).spoils;
-            const time_duration time_left = shelf_life - shelf_life * relative_rot;
+            time_duration time_left = shelf_life - shelf_life * relative_rot;
 
-            const int skill = p.get_skill_level( skill_survival );
             const float days_left = to_days<float>( time_left );
             if( days_left <= 0.25 ) {
                 if( item->is_going_bad() ) {
                     return _( "soon!" );
                 } else {
-                    const time_duration time_left = time_duration::from_hours( 6 );
+                    time_left = time_duration::from_hours( 6 );
                 }
             } else if( days_left <= 7 ) {
-                const time_duration time_left = time_duration::from_days( ceilf( days_left ) );
+                time_left = time_duration::from_days( ceilf( days_left ) );
             } else {
-                const time_duration time_left = time_duration::from_days( ceilf( days_left / 7 ) * 7 );
+                time_left = time_duration::from_days( ceilf( days_left / 7 ) * 7 );
             }
             if( time_left > get_edible_comestible( loc ).spoils ) {
-                const time_duration time_left = get_edible_comestible( loc ).spoils;
+                time_left = get_edible_comestible( loc ).spoils;
             }
             return to_string_clipped( time_left );
         }

